sample/src/main/java/com/manaschaudhari/android_mvvm/sample/ItemViewModel.java

```diff
@@ -25,7 +25,7 @@
 
 import rx.functions.Action0;
 
-public class ItemViewModel implements ViewModel {
+public class ItemViewModel extends ViewModel {
```

Comment:
Why switch to inheritance? Interfaces allow for more flexibility in building ViewModels.